### ISSUE TITLE
L&R analogue triggers are not working properly in NBA 2k1 and 2k2

### ISSUE THREAD

**Issue by helcril** [twinaphex/reicast-emulator#1120]
Gamepad Thrustmaster Run'N'Drive Wireless. 
I managed to configure triggers, but they do not work correct. When I perform a freethrow and push triggers even with a good player their movement is very jerky. Though it might be smooth if player is good at freethrows. Deadband is set to 0%, so that's not the issue.
Can I somehow configure analogue triggers to work as they should?

**Comment by flyinghead** [comment]
Please install and run this CDI: [ControllerTest](https://github.com/libretro/libretro-content/blob/master/Sega%20-%20Dreamcast/ControllerTest-DJ.cdi)
Are your triggers working correctly with it?

**Comment by helcril** [comment]
Thank you for quick reply!
Yes in this test triggers work almost perfect. When I push them purple circles change their size pretty smoothly.
But they are not visible, when triggers are pushed full and start getting bigger as I release triggers bit by bit. Is this correct work?

**Comment by flyinghead** [comment]
This is not correct: the circles should get bigger as you push the triggers. When pushed fully the circles should have their maximum size and should get smaller when you release the triggers.
However, it's probably an issue with either RetroArch or your gamepad.

Make sure the Flycast core option "Digital Triggers" is off. 
Check your controller bindings in Settings > Input to make sure you don't have multiple buttons/axes bound to the triggers. Some gamepad report trigger presses both as digital and analog so this can confuse RetroArch.

It would also help to test analog triggers with a different core to make sure it's not a core-specific issue.

**Comment by helcril** [comment]
I managed to configure triggers that they work correct. The more I push them, the bigger circles become. However sensibility is too high for NBA 2K games.
There is one solution for free throws I found. If analog sensibility is set to 0.5 this is optimal for freethrow mode, but it is to small for speeding up a player during his run.
Is there any option to map analogue triggers only for throwing with 0.5 sensibility, plus mapping digital bumpers for 100% trigger value?

**Comment by flyinghead** [comment]
You should probably look at RetroArch input settings. Flycast just gets the triggers values from RetroArch.
